docs/guides/advanced/testing.md

```diff
@@ -1,5 +1,10 @@
 # Testing with Logfire
 
+!!! tip "Running under Pytest... 🧪"
+    When running your test suite under Pytest, we set [`send_to_logfire=False`][logfire.configure(send_to_logfire)] by default.
+
+    Which means that your logs and spans will not be sent to the Logfire service, but instead captured in memory.
```

Comment:
I find this line weird, it sounds like it's saying they're captured somewhere in memory by default, not that capfire does that.

Reply:
better?